applications/directory/src/components/ProfileReviewComplete.js

```diff
@@ -0,0 +1,49 @@
+import Card from '@mui/material/Card';
+
+const ProfileReviewComplete = () => {
+  return (
+    <Card
+      sx={{
+        mr: ['0', '5%'],
+        mb: ['5%', '0'],
+        flex: 1,
+        textAlign: 'center',
+        width: '100%',
+        '& p': {
```

Comment:
CSS for elements should not be nested. All new code must have the styling assigned to the specific element.

Use `Box` + `sx` below.

Reply:
Not worked with styles here, just copied from another component. It will be all done separately in a different branch